packages/wdio-cucumber-framework/src/reporter.ts

```diff
@@ -141,9 +141,14 @@ class CucumberReporter {
             state = 'pass'
             break
         case Status.PENDING:
+            state = 'pending'
+            break
         case Status.SKIPPED:
+            state = 'skip'
```

Comment:
This change breaks wdio-report, because wdio-report does not listen on event "test:skip", it only listen on event "test:pending" for now.

Reply:
Sorry for the typo. Should be wdio-reporter. I've filed a bug https://github.com/webdriverio/webdriverio/issues/8007